src/fides/api/alembic/migrations/versions/a1e23b70f2b2_add_notice_and_exp_translations.py

```diff
@@ -220,24 +231,26 @@ def upgrade():
     )
     op.add_column(
         "privacyexperienceconfighistory",
-        sa.Column("dismissable", sa.Boolean(), server_default="t", nullable=False),
+        sa.Column("dismissable", sa.Boolean(), nullable=True),
```

Comment:
Reverting PrivacyExperienceConfigHistory.dismissable to be nullable, so this isn't backfilled on historical records because this could be inaccurate on historical records created before this release.  This value will be set going forward when experience config history records are created.